sdk/keyvault/ci.yml

```diff
@@ -33,6 +33,8 @@ pr:
     exclude:
       - sdk/keyvault/microsoft-azure
 
+
+
 stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```

Comment:
Does the CI auto detect which tests to run (Track 1 or Track 2) based on code changes ?

Reply:
I'll be doing some further tweaks to the path filters.

Reply:
We have two pipeline definitions and two yaml files. Each pipeline runs the appropriate set of tests. If you change files in both track 1 and track 2 -- two separate pipelines will run but I expect this to be the rare case. In fact the track 1 builds I think will be temporary.